lib/srv/db/sqlserver/connect_test.go

```diff
@@ -16,19 +16,28 @@ package sqlserver
 
 import (
 	"context"
+	_ "embed"
+	"errors"
 	"os"
+	"os/exec"
+	"path/filepath"
 	"testing"
 	"time"
 
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 
+	"github.com/gravitational/teleport/api/client/proto"
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/services"
 	"github.com/gravitational/teleport/lib/srv/db/common"
 	"github.com/gravitational/teleport/lib/srv/db/sqlserver/protocol"
 )
 
+//go:embed testdata/kinit.cache
```

Comment:
Nit do we need `testdata/kinit.cache` duplication in `lib/srv/db/sqlserver/kinit/testdata/kinit.cache` and `lib/srv/db/sqlserver/testdata/kinit.cache` 

Can we just embed `kinit/testdata/kinit.cache` ?